src/wallet/rpcwallet.cpp

```diff
@@ -31,7 +31,14 @@
 
 CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
 {
-    // TODO: Some way to access secondary wallets
+    if (!request.wallet.empty()) {
+        for (const auto& wallet : ::vpwallets) {
```

Comment:
Sure! When that happens proper concurrency protection must be added no? IMO is work for other PR.